### PR TITLE
Small Fix: Safety net for Requests on things that aren't url

### PR DESCRIPTION
**Pull request by maxiin**
So I wanted to try the pluging with Discord and when I tried to put my token on the "webhook token" field it crashed my game; I took a look at the code and figured out that it wanted the whole url for discord webhooks, so I updated that part of the UI to "webhooks URL" and put a few checks on the Deliveries (Discord & Ntfy) so it tests for a valid url before setting the delivery as active, I tested with a few custom urls (selfhosted & Base Ntfy, discord webhooks) and it seems to work fine! Also added a Argument Exception just in case.
Thanks for the Plugin! Helps a lot 😃

### CODE REVIEW

**Comment by maxiin** [comment]
Also, just curious: Why does the Dalamud installer version not have Ntfy? its at 1.3.0.0 if it helps

**Comment by lostkagamine** [comment]
> Also, just curious: Why does the Dalamud installer version not have Ntfy? its at 1.3.0.0 if it helps

It... definitely should! Maybe I forgot to update the repository after the merge, but it should...

**Comment by lostkagamine** [comment]
So turns out I had hallucinated sending the patch into the main plugin repository. So it wasn't there. Oops. It's there now; I only noticed with 7.11's API level bump. Sorry!!! Let me go get this reviewed for you.

File: PushyFinder/Delivery/DiscordDelivery.cs [hunk 1]
```diff
@@ -8,7 +9,8 @@ namespace PushyFinder.Delivery;
 
 internal class DiscordDelivery : IDelivery
 {
-    public bool IsActive => !Plugin.Configuration.DiscordWebhookToken.IsNullOrWhitespace();
+    public bool IsActive => !Plugin.Configuration.DiscordWebhookToken.IsNullOrWhitespace() && 
+                            Uri.IsWellFormedUriString(Plugin.Configuration.DiscordWebhookToken, UriKind.Absolute);
```
Comment (lostkagamine) [inline PushyFinder/Delivery/DiscordDelivery.cs:13]: Nice

File: PushyFinder/Delivery/NtfyDelivery.cs [hunk 2]
```diff
@@ -17,7 +19,7 @@ public void Deliver(string title, string text)
 
     private static async void DeliverAsync(string title, string text)
     {
-        var args = new Dictionary<string, string>
+            var args = new Dictionary<string, string>
```
Comment (lostkagamine) [inline PushyFinder/Delivery/NtfyDelivery.cs:22]: Why the whitespace?

File: PushyFinder/Windows/ConfigWindow.cs [hunk 3]
```diff
@@ -60,7 +60,7 @@ private void DrawDiscordConfig()
     {
         {
             var cfg = Configuration.DiscordWebhookToken;
-            if (ImGui.InputText("Webhook token", ref cfg, 2048u)) Configuration.DiscordWebhookToken = cfg;
+            if (ImGui.InputText("Webhook URL", ref cfg, 2048u)) Configuration.DiscordWebhookToken = cfg;
```
Comment (lostkagamine) [inline PushyFinder/Windows/ConfigWindow.cs:63]: Good catch

**Review by lostkagamine — APPROVED** [review]

File: PushyFinder/Delivery/NtfyDelivery.cs (hunk 2 above)
Comment (maxiin) [reply PushyFinder/Delivery/NtfyDelivery.cs:22]: No reason, just a mess up xD

**Pull request merged**